README.md

```diff
@@ -1 +1,154 @@
+
+  - [Installation](#installation)
+  - [Including in your project](#using)
+  - [TCModel](#tcmodel) - Creates a stateful model to store a Transparency and Consent user interaction with all the fields specified in the [TC string](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#about-the-transparency--consent-string-tc-string) encoding schema.
+     - [Information that is stored in a TC string (specification)](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#what-information-is-stored-in-a-tc-string)
+- [GVL](#gvl) - The [Global Vendor List](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#the-global-vendor-list) contains all of the information about vendors and legal language to display to users in a CMP user interface, this component helps manage it.
+- [TCModelEnum](#tcmodelenum) - The library class for implementing the dynamic getting and setting fields of the [TCModel](#tcmodel) during runtime.
+- [TCString](#tcstring) - Decodes a [TC string](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20Consent%20string%20and%20vendor%20list%20formats%20v2.md#about-the-transparency--consent-string-tc-string) into a [`TCModel`](#tcmodel)
+      + [Decode an IAB TC String](#using)
+  
+#### Installation
+- Prerequisites
+    + Java 1.8 
+    + Maven 2.0+
+
+Clone the project into your local repository. Run the below command from the project directory to install the project locally.
+```
+mvn clean install
+```
+
+#### Using
+```java
+import com.iabtcf.TCModel;
+import com.iabtcf.TCString;
+public class IabTCFCore {
+    // Sample TC STring to be decoded
+    private String base64TcString = "";
+
+    @Test
+    public void readTcString() {
+        TCString tcString = TCString.getInstance();
+        // take an encoded TC string and decode into a TCModel
+        final TCModel tcModel = tcString.decode(base64TcString);
+    }
+}
+```
+# TCModel
+
+[API Docs](https://www.iabtcf.com/api/core/classes/tcmodel.html)
+
+### Creating a new TCModel
```

Comment:
Yes! Understood. I just added them in case someone wants to add the encoder later, So he/she might have an idea about the TCModel class. If required, I will remove them.

Reply:
I would remove them.  It's going to cause more problems when people open issues on the repository saying "The documentation says I can do this but it doesn't work."  So the documentation should reflect only what is currently available.